### PR TITLE
Feature/solcx binary path

### PR DESCRIPTION
**Pull request by MatthiasLohr**
### What was wrong / missing?

  * There was an option missing for configuring/overwriting solc binary installation path

### How was it fixed / added?

You can now configure the path either using an environment variable `SOLCX_BINARY_PATH` or by providing an additional parameter to install method.

#### Cute Animal Picture

Sorry, don't have one.

### CODE REVIEW

**Comment by MatthiasLohr** [comment]
Mh, ok, I think I missed the part for providing the path to the `compile` methods...

**Comment by iamdefinitelyahuman** [comment]
Looks good overall, can you fix the linting issues tho?

**Comment by MatthiasLohr** [comment]
Done :)

File: solcx/install.py [hunk 1]
```diff
@@ -370,10 +381,8 @@ def _install_solc_osx(version, allow_osx, show_progress):
...
 
 if __name__ == "__main__":
-    try:
-        version = sys.argv[1]
-    except IndexError:
-        LOGGER.error("Invocation error.  Should be invoked as `./install_solc.py <release-tag>`")
-        sys.exit(1)
-
-    install_solc(version)
+    argument_parser = argparse.ArgumentParser()
```
Comment (iamdefinitelyahuman) [inline solcx/install.py]: Thank you for swapping this to `argparse` as well, it was long overdue.

Comment (MatthiasLohr) [reply solcx/install.py]: Welcome :)

**Comment by MatthiasLohr** [comment]
Thanks for merging :)

**Comment by iamdefinitelyahuman** [comment]
Thanks for the contribution!  I'll have a new release out later today.

**Comment by MatthiasLohr** [comment]
That would be great, thank you so much!

**Comment by iamdefinitelyahuman** [comment]
[v0.8.0](https://github.com/iamdefinitelyahuman/py-solc-x/releases/tag/v0.8.0) is released and available on pypi :) thanks again for your contribution!

**Pull request merged**